code/game/objects/effects/overlays.dm

```diff
@@ -325,15 +332,15 @@
 	var/mob/user
 	var/damage = 20
 
-/obj/effect/overlay/temp/ratvar/volt_hit/New(loc, caster, multiplier)
+/obj/effect/overlay/temp/ratvar/volt_hit/Initialize(mapload, caster, multiplier)
 	if(multiplier)
 		damage *= multiplier
 	duration = max(round(damage * 0.2), 1)
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?